amgut/db/scrub.sql

```diff
@@ -0,0 +1,343 @@
+-- This script is used to scrub the live American Gut database in order to
+-- generate a test database for Travis without personal information but
+-- representative of the real world.
+
+-- Function to generate a string with all the different characters present in
+-- a column (adapted from: http://dba.stackexchange.com/a/83679)
+CREATE FUNCTION retrieve_chars(column_name varchar, table_name varchar) RETURNS varchar AS $$
```

Comment:
The database should be configured as UTF8 so non-utf8 characters will fail to be inserted in the DB directly, right?